Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang
```

Comment:
Doesn't this make a container which contains the go compiler and the go source code?

That is more than is needed to run rclone, rclone can run from the scratch container, but ideally would have some SSL roots in too.

There is an example of how to do that here (scroll down to Dockerfile.multistage): https://www.callicoder.com/docker-golang-image-container-example/

Reply:
Yes, it does.
I've used multi-stage builds in some cases, where size is priority. However, it adds [a slighty] more time to the build and Dockerfile complexity. Most of time, the tradeoff in the Dockerfile is the convenience as a developer tool. Usually developers focus on ease of use, efficency in repeating the build a lot and having debug tools at hand, whereas distribution usage priorize image size.
I've done it this way, since it's the middle spot, IMO. Since, I'm not as familiar with Golang development to have all considerations in mind, I can switch to a multi-stage build if you think it's better.

Reply:
I think a multi-stage build would be my preference.  The golang official image is 800 MB wheras a bare one should be ~10MB which is enough of a win in my mind to make it worth doing :-)

Reply:
Pushed new multi-staged build Dockerfile.
Tested it, and the single-stage vs ms build is (29s vs 33s), not a big deal.
We'll see if it makes local development workflow much worse or not. I can't say for sure.
Hopefully, I'll be able to try it myself with some easy issue, if time permits. :)